src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -339,19 +341,38 @@ private void notifyUser(MediaRequestEntity mediaRequestEntity,
             if (notificationTemplateName.equals(NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString())) {
 
                 List<DefendantEntity> defendantList = mediaRequestEntity.getHearing().getCourtCase().getDefendantList();
+
                 List<String> defendantNames = new ArrayList<>();
-                for (DefendantEntity defendant : defendantList) {
-                    defendantNames.add(defendant.getName());
+                String defendants = NO_DEFENDANTS;
+                if (!defendantList.isEmpty()) {
+                    for (DefendantEntity defendant : defendantList) {
+                        defendantNames.add(defendant.getName());
+
+                    }
+
+                    defendants = defendantNames.stream().collect(Collectors.joining(","));
+                }
+
+                String courthouseName = mediaRequestEntity.getHearing().getCourtCase().getCourthouse().getCourthouseName();
+                String hearingDate = "";
```

Comment:
Would this and the block below be better as a ternary operator? e.g.

`String hearingDate = mediaRequestEntity.getHearing().getHearingDate() != null ? String.valueOf(mediaRequestEntity.getHearing().getHearingDate()) : NOT_APPLICABLE;
`